### DISCUSSION TITLE
A little help, please?

### DISCUSSION THREAD

**Discussion by deadhammerhead** [TPC-Council/HammerDB#526] [Q&A] [answered]
The docs are often quite verbose -- so I may have missed something here.

When I connect to HammerDB and start the virtual users am I supposed to be
simulating some sort of traffic and pumping it into the database?

Thanks!

**Comment by sm-shaw** [comment, marked as answer]
The Quick Start in the Documentation is [here](https://www.hammerdb.com/docs/ch02.html) 
That is 5 pages step by step with 1 sentence and a screenshot per step of what you need to do to answer your question.
